pkg/transforms/genericResourceConfig.go

```diff
@@ -126,10 +126,19 @@ var defaultTransformConfig = map[string]ResourceConfig{
 		properties: []ExtractProperty{
 			{Name: "ready", JSONPath: `{.status.conditions[?(@.type=='Ready')].status}`},
 			{Name: "status", JSONPath: `{.status.conditions[?(@.type=='Ready')].reason}`},
+			{Name: "phase",  JSONPath: `{.status.phase}`},
 			{Name: "indications", JSONPath: `{.status.indications}`}, // this is an array of strings - will collect array items separated by ;
 			{Name: "sourceVM", JSONPath: `{.spec.source.name}`},
 		},
 	},
+	"VirtualMachineRestore.snapshot.kubevirt.io": {
+		properties: []ExtractProperty{
+			{Name: "ready", JSONPath: `{.status.conditions[?(@.type=='Ready')].status}`},
+			{Name: "status", JSONPath: `{.status.conditions[?(@.type=='Ready')].reason}`},
+			{Name: "lastRestoreTime", JSONPath: `{.status.restoreTime}`},
```

Comment:
This will get collected as a string. I'm not sure if it will work the same as other properties with time. We'd need to test it.

Reply:
I have tested in my local env with the PR image. I see the search results with the string as I would expect:
```
                   {
                        ...
                        "kind": "VirtualMachineRestore",
                        "lastRestoreTime": "2025-04-29T18:50:05Z",
                        ...
                    }
```

Reply:
Are you able to search like other dates from the UI? Like `lastRestoreTime: <day`

Reply:
<img width="493" alt="Screenshot 2025-05-07 at 2 53 47 PM" src="https://github.com/user-attachments/assets/544a7bbc-4a34-4588-8360-1379dc1278b0" />

Reply:
Thanks for confirming. Nice expandable UI code! 🥇